src/app/layout-theme-toggle/layout-theme-toggle.component.scss

```diff
@@ -10,4 +10,7 @@
     margin: 0;
   }
 
+  .theme_radio {
```

Comment:
We usually use dashes to separate words. And let's move all rules out of `host` as there is no reason to nest it since the component has style encapsulation enabled.